plugins/connectrpc/dart/v0.1.0/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.12
+FROM dart:3.5.4-sdk AS build
+
+WORKDIR /build
+RUN curl -fsSL https://github.com/connectrpc/connect-dart/archive/refs/tags/v0.1.0.tar.gz --compressed -o connect-dart.tar.gz \
```

Comment:
We typically `RUN git clone --depth 1` instead of fetching the .zip, any reason we have to deviate here from the other Dockerfiles?

Reply:
This is similar to the dart base plugin image, we just swapped the url from pub.dev to github and cd into the directory. I can change it to git clone if that is better suited

Reply:
Would prefer if we used the `git clone` approach.